sound/hybrid_font.h

```diff
@@ -120,7 +130,49 @@ class HybridFont : public SaberBase {
       PlayPolyphonic(effect);
     }
   }
+  void StartSwing() override {
+    if (!guess_monophonic_) {
+      if (!swing_player_) {
+        swing_player_ = PlayPolyphonic(&swng);
+      } else {
+        second_swing_player_ = PlayPolyphonic(&swng);
+      }
+    } else {
+      PlayMonophonic(&swing, &hum);
+    }
+  }
+
+  float SetSwingVolume(float swing_strength, float mixhum) {
+    if (IsSwingPlaying()) {
+      float accent_volume = powf(swing_strength, config_.ProffieOSSwingVolumeSharpness) * config_.ProffieOSMaxSwingVolume;
+      swing_player_->set_volume(accent_volume);
+      if (second_swing_player_) {
+        second_swing_player_->set_volume(accent_volume);
+      }
+      mixhum = mixhum * config_.ProffieOSMaxSwingDucking;
```

Comment:
We really need to make the ducking larger when the accent_volume is larger.
Unconditionally ducking by a fixed amount while the swing is playing is not going to sound good.
It might be better to just not duck at all and let the dynamic mixer sort it out, but at least that is configurable right now.

Reply:
removed ducking, not required.